spp_event_data/wizard/create_event_wizard.py

```diff
@@ -47,7 +47,7 @@ def next_page(self):
                 wiz = self.env[wizard_model].create({"event_id": event_id.id})
 
                 return {
-                    "name": _("Create %s Wizard" % view_name),
+                    "name": _("Create %s Wizard") % view_name,
```

Comment:
Correct format is:
> _("Create %s Wizard", view_name)

https://www.odoo.com/documentation/15.0/developer/howtos/translations.html?highlight=translation#variables